src/components/image-content-block/image-content-block.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { Trans } from 'react-i18next';
+import styled, { css } from 'styled-components';
+import { up } from '../breakpoint/breakpoint';
+import { theme } from '../layout/theme';
+import { TextStyles } from '../typography/typography-v2';
+
+const mapTextAlignToFlex = (textAlign: 'right' | 'left' | 'bottom') => {
+  if (textAlign === 'right') {
+    return 'row';
+  } else if (textAlign === 'left') {
+    return 'row-reverse';
```

Comment:
I mentioned the `order` attribute which is much more explicit than just reversing everything. This works here as we only have two elements but remember the `order` property to swap children to arbitrary places in the flex layout 👍

Reply:
Okay, yes, this way felt quite cool to me, since I could also o the bottom thing within.
But I will read about `order` for the next time.